calibr8/contrib/normal.py

```diff
@@ -0,0 +1,256 @@
+"""
+This module implements reusable calibration models
+with Normal distributions for the dependent variable.
+"""
+from typing import Optional, Tuple
+
+from . import noise
+from .. import core
+
+
+class BasePolynomialModelN(core.ContinuousUnivariateModel, noise.NormalNoise):
+    def __init__(
+        self, *,
+        independent_key: str, dependent_key: str,
+        mu_degree: int, sigma_degree: int=0,
+        theta_names: Optional[Tuple[str]]=None,
+    ):
+        """ Template for a model with polynomial trend (mu) and sigma (as a function of mu).
```

Comment:
This descirption could be valid for the Student-t version as well. Should we not mention the Normal distribution in the description?